src/storage/index.js

```diff
@@ -3,3 +3,4 @@
 import createWebStorage from './createWebStorage'
 
 export default createWebStorage('local')
+export { default as session } from './session'
```

Comment:
I plan to add `#__PURE__` annotations with my plugin - https://github.com/Andarist/babel-plugin-annotate-pure-calls . So unused version should get dropped. The goal for me here was to provide a single `redux-persist/storage` entry